agenthub/codeact_agent/codeact_agent.py

```diff
@@ -203,9 +203,10 @@ def step(self, state: State) -> Action:
         }
         try:
             response = self.llm.completion(**params)
-        except Exception:
+        except Exception as e:
+            error_message = '{}: {}'.format(type(e).__name__, str(e).split('\n')[0])
             return AgentFinishAction(
-                thought='Agent encountered an error while processing the last action. Please try again.'
+                thought=f'Agent encountered an error while processing the last action.\nError: {error_message}\nPlease try again.'
```

Comment:
This message gets sent in the chatbox, right? I guess it would be better in the status bar in the middle... But it's an improvement, thanks!

Reply:
Yeh, not gonna touch frontend code here. 😬